### ISSUE TITLE
Aligning with pre-trained models

### ISSUE THREAD

**Issue by AmitMY** [sign/image-latent-transformer#7]
For training, we can initialize the model from scratch, or from pretrained checkpoints of various components.

https://github.com/sign/image-latent-transformer/blob/9e14bd2aa6ea00b8a00f13369e5633036c9c00ae/image_latent_transformer/ilt.py#L41

If we do `load_pretrained=True`, it might be useful to start the training with all components frozen except for:

1. The embedding layers (`bytes_encoder`, `bytes_decoder`)
2. The LM Head (`bytes_decoder`)
3. The mapping layers (`encoder_mapping`, `decoder_mapping`)

https://github.com/sign/image-latent-transformer/blob/9e14bd2aa6ea00b8a00f13369e5633036c9c00ae/image_latent_transformer/ilt.py#L87-L88

That way, we don't immediately destroy the already "good" pretrained models.

**Comment by AmitMY** [comment]
Here's an example loss curve, with a `--warmup_freeze_steps 5000`
<img width="449" height="282" alt="Image" src="https://github.com/user-attachments/assets/442d5be1-5ad8-4444-b7ea-358ef5ef4da3" />
I would prefer the loss curve to be smoother around the 5k point, and not explode like that.
